src/pages/index.tsx

```diff
@@ -2,65 +2,61 @@ import React, { FunctionComponent } from "react";
 
 const Home: FunctionComponent = () => {
   return (
-    <div className={"govuk-width-container"}>
-      <main className={"govuk-main-wrapper"}>
-        <h1 className="govuk-heading-l">Beacon registration service</h1>
+    <>
```

Comment:
Assume this whole block will be removed/updated once we pick up the [service start page](https://trello.com/c/lMvXm62L/178-create-service-start-page-skeleton-for-registering-a-beacon)?